doc/Type/Metamodel/Stashing.pod6

```diff
@@ -0,0 +1,61 @@
+=begin pod :kind<Type> :subkind<role> :category<metamodel>
+
+=TITLE role Metamodel::Stashing
+
+=SUBTITLE Metarole for type stashes
+
+    role Metamodel::Stashing { }
+
+Types may have a stash associated with them. For types where this is
+the case, they can be used like namespaces; you can get their stash
+using their C<WHO> property or the C<::> dispatch operator:
+
+=begin code :lang<perl6>
+module Nested {
+    module Namespace {
+        constant Symbol = $?MODULE;
+    }
+}
+
+say Nested::Namespace::Symbol;         # OUTPUT: «(Namespace)␤»
+say Nested.WHO<Namespace>.WHO<Symbol>; # OUTPUT: «(Namespace)␤»
+=end code
+
+C<Metamodel::Stashing> is the metarole that handles creating and
+setting a stash object for types. Types used with this metarole are
+expected to support naming, so when writing custom HOWs that do it,
+ensure they also do L<Metamodel::Naming|/type/Metamodel::Naming>.
+
+I<Warning>: This role is part of the Rakudo implementation, not a part
```

Comment:
Maybe put this on top.

Reply:
I put it here because it's put in the same place in the other `Metamodel` pages. I can move it for all of them after this if it'd fit better on top.

Reply:
I think so...